app/lib/response_util.py

```diff
@@ -1,3 +1,16 @@
+def _returnUser(user):
```

Comment:
I think in your other PR about sessions I made a comment about pushing this logic into the model itself. Feel free to do that here or in the other PR.